src/index.ts

```diff
@@ -6,6 +6,7 @@ export {
 	setCookie,
 	setSessionCookie,
 } from './cookies';
+export type { Country, CountryCode } from './countries';
```

Comment:
Should `Country` be exported as a type here?

Reply:
after chatting offline, no :)